### ISSUE TITLE
There are no man-pages for some functions published in auparse.h and libaudit.h

### ISSUE THREAD

**Issue by miztake** [linux-audit/audit-userspace#193]
Among the functions published in auparse.h and libaudit.h,
there are no man-pages for the following functions.
If there are any functions that shouldn't be published below,
they should be removed from auparse.h and libaudit.h.

- auparse.h
auparse_get_record_interpretations

- libaudit.h
audit_close
audit_determine_machine
audit_format_signal_info
audit_name_to_field
audit_field_to_name
audit_name_to_syscall
audit_syscall_to_name
audit_name_to_flag
audit_flag_to_name
audit_name_to_action
audit_action_to_name
audit_name_to_msg_type
audit_msg_type_to_name
audit_name_to_machine
audit_machine_to_name
audit_machine_to_elf
audit_elf_to_machine
audit_operator_to_symbol
audit_name_to_errno
audit_errno_to_name
audit_name_to_ftype
audit_ftype_to_name
audit_name_to_fstype
audit_fstype_to_name
audit_number_to_errmsg
audit_is_enabled
audit_request_features
audit_get_features
audit_reset_lost
audit_reset_backlog_wait_time_actual
audit_set_feature
audit_set_loginuid_immutable
audit_add_dir
audit_add_watch_dir
audit_trim_subtrees
audit_make_equivalent
audit_value_needs_encoding
audit_encode_value
audit_rule_create_data
audit_rule_init_data
audit_rule_syscall_data
audit_rule_syscallbyname_data
audit_rule_fieldpair_data
audit_rule_interfield_comp_data
audit_rule_free_data
audit_can_control
audit_can_write
audit_can_read

**Comment by stevegrubb** [comment]
This is true and intentional. Many of these functions are related to generating machine readable rules from human readable text. They are not as likely to be used by someone writing utilities for the audit logs. Of those, audit_value_needs_encoding, audit_encode_value, and audit_close might be of interest to programmers. And at a lower priority, I'd say audit_is_enabled,
audit_request_features, audit_get_features, audit_reset_lost, audit_reset_backlog_wait_time_actual, audit_set_feature, audit_set_loginuid_immutable, audit_can_control, audit_can_write, and audit_can_read. The audit_can* could probably be 1 man page. I'm wondering if some of the others could also be combined or added to exist pages?

**Comment by miztake** [comment]
Dear @stevegrubb,

Thank you for your reply.

> They are not as likely to be used by someone writing utilities for the audit logs.

Understood.

> Of those, audit_value_needs_encoding, audit_encode_value, and audit_close might be of interest to programmers.

I agree with this opinion.
First, I'll create these man-pages.

> And at a lower priority, I'd say ...

I understand that these are low priorities.

> The audit_can* could probably be 1 man page.

I agree that audit_can* could probably be 1 man page.

> I'm wondering if some of the others could also be combined or added to exist pages?

I don't think it is necessary to combine or add other pages.

Regards,

**Comment by stevegrubb** [comment]
Commit 25d5fea1 addresses the functions listed in my previous comment. At this time, I don't think we will address the others. They are too esoteric to be of general interest. Thank for reporting the issue.
